Makefile

```diff
@@ -390,6 +405,10 @@ docker-push:
 		docker push $(DOCKER_NAMESPACE)/jaeger-$$component ; \
 	done
 
+.PHONY: dockerx-push
+dockerx-push: setup-buildx-builder
+	BUILDX_PUSH=true $(MAKE) docker-images-only
```

Comment:
Var before command becomes env var, after command it becomes make var. Do you want `$(MAKE) docker-images-only BUILDX_PUSH=true`?

Reply:
env var could overwrites make var `BUILDX_PUSH` when define with `?=`.
will same as `$(MAKE) docker-images-only BUILDX_PUSH=true`.

i will update this for style.